### PR TITLE
Added enable searching of posts and fixed typos

### PR DESCRIPTION
**Pull request by JeffSchering**
This is for [Adding the ability to disable full text search queries for master #6102](https://github.com/mattermost/platform/pull/6102)

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2320,9 +2320,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with whole number input.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Searching of Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This setting enables the ability for users to search messages. If not enabled, when users attempt to use the search box they get an error message.
```
Comment (lindy65) [inline source/administration/config-settings.rst]: Thought about this paragraph for quite a while and, to me, it just doesn't read smoothly...

How about:
`If this setting is enabled, users can search messages.  If it's disabled, users get an error message when they attempt to use the search box.`

Comment (JeffSchering) [inline source/administration/config-settings.rst]: Done. Thanks!

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2320,9 +2320,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with whole number input.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Searching of Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If this setting is enabled, users can search messages. If it's disabled, users get an error message when they attempt to use the search box.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: 1) Maybe add a note it's for improving performance?

E.g. if lots of users do searching in a deployment with millions of messages, it has the potential of causing the server to slow down

2) Also propose adding a note that if you're looking to improve performance, you can first try to use the search read replicas, then disable search

Comment (JeffSchering) [inline source/administration/config-settings.rst]: 1. Ok. But it seems a bit odd... "If the feature is too popular, turn it off." :)
2. I'm not sure about mentioning search replicas. They are only available in E20 with a cluster config.

**Pull request merged**